build/kube_setup.sh

```diff
@@ -93,7 +94,14 @@ function install_strimzi_operator {
     [[ $PLATFORM == "Darwin" ]] && sed -i '' "s/namespace: .*/namespace: ${STRIMZI_OPERATOR_NS}/" *RoleBinding*.yaml \
         || sed -i "s/namespace: .*/namespace: ${STRIMZI_OPERATOR_NS}/" *RoleBinding*.yaml
 
-    $ROOT_DIR/build/fix_namespace.py 060-Deployment-strimzi-cluster-operator.yaml "$WATCH_NS"
+    if ! test -f $ROOT_DIR/build/${FIX_NAMESPACE_SCRIPT}; then
```

Comment:
Should we just always download it?

Otherwise the other problem we run into is: could someone have a very old `fix_namespace.py` sitting in the dir that never gets updated? :)

Reply:
Agreed on this